pkg/util/proto/marshal_any_test.go

```diff
@@ -0,0 +1,27 @@
+package proto_test
+
+import (
+	"github.com/golang/protobuf/proto"
+
+	util_proto "github.com/Kong/kuma/pkg/util/proto"
+	"github.com/Kong/kuma/pkg/xds/envoy"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("MarshalAnyDeterministic", func() {
+	It("should marshal deterministically", func() {
+		tags := map[string]string{
+			"service": "backend",
+			"version": "v1",
+			"cloud":   "aws",
```

Comment:
Should we increase the number of tags to have higher chances to get a difference (eventually)?

Reply:
The problem was very visible with 100 iterations with those 3 of tags.